scarpe-components/lib/scarpe/components/calzini.rb

```diff
@@ -126,8 +126,14 @@ def drawable_style(props)
     styles[:left] = dimensions_length(props["left"]) if props["left"]
     styles[:width] = dimensions_length(props["width"]) if props["width"]
     styles[:height] = dimensions_length(props["height"]) if props["height"]
+    styles[:"margin"] = dimensions_length(props["margin"]) if props["margin"]
```

Comment:
Minor thing, but I think we can get rid of the "margin" style here now. Certainly it shouldn't need or use props["margin"].